File: compiler-solo-typechecker/src/test/resources/IntGreaterThanOrEqual.java
public class IntGreaterThanOrEqual {
    public static void main(String[] args) {
        StaticJavaLib.assertTrue(2 >= 3 == false);
        StaticJavaLib.assertTrue(3 >= 2 == true);
        StaticJavaLib.assertTrue(2 >= 2 == true);

        StaticJavaLib.assertTrue(greaterThanOrEqual(2, 3) == false);
        StaticJavaLib.assertTrue(greaterThanOrEqual(3, 2) == true);
        StaticJavaLib.assertTrue(greaterThanOrEqual(2, 2) == true);
    }   
    
    static boolean greaterThanOrEqual(int i, int j) {
        return i >= j;
    }
}
    

File: compiler-group-traslator/src/test/resources/c/IntGreaterThanOrEqual.c
#include<stdio.h>
#include<stdlib.h>
#include<stdbool.h>
#include "GC.c"
#include "ExceptionHandlingFunctions.c"
VM* vm;
bool greaterThanOrEqual(int i, int j) {
	void* RU = i >= j;
	return RU;

	//set all vars to zero and have the TEMP variable for the return expression
}
int main(char* args) {
	vm = newVM();
	assertTrue(2 >= 3 == false);
	assertTrue(3 >= 2 == true);
	assertTrue(2 >= 2 == true);
	assertTrue(greaterThanOrEqual(2, 3) == false);
	assertTrue(greaterThanOrEqual(3, 2) == true);
	assertTrue(greaterThanOrEqual(2, 2) == true);
	return 0;
}